virttest/utils_v2v.py

```diff
@@ -718,6 +723,23 @@ def v2v_cmd(params):
 
     # Construct a final virt-v2v command
     cmd = '%s %s' % (V2V_EXEC, options)
+    logging.debug(cmd)
+
+    # To deal with inputs during conversion
+    if custom_inputs or (sasl_user and sasl_pwd):
+        v2v_log = os.path.join(data_dir.get_tmp_dir(), 'v2v_log')
+        v2v_session = aexpect.ShellSession(command=cmd,
+                                           output_func=utils_misc.log_line,
+                                           output_params=(v2v_log,))
+        remote.handle_v2v_prompts(v2v_session, sasl_user, sasl_pwd, timeout=v2v_timeout/2,
```

Comment:
It's better to close the session when meets some exception like the way in remote_login function.
try:
        handle_prompts(session, username, password, prompt, timeout)
    except Exception:
        session.close()
        raise